src/mgmt/tests/mocks.h

```diff
@@ -78,4 +79,20 @@ extern void		*ds_mgmt_pool_delete_acl_result_ptr;
 void mock_ds_mgmt_pool_delete_acl_setup(void);
 void mock_ds_mgmt_pool_delete_acl_teardown(void);
 
+/*
+ * Mock ds_mgmt_list_pools
+ */
+extern int				ds_mgmt_list_pools_return;
+extern char				ds_mgmt_list_pools_group[];
+extern void				*ds_mgmt_list_pools_npools_ptr;
+extern uint64_t				ds_mgmt_list_pools_npools;
```

Comment:
nit: indented one tab too far?

Reply:
Same as above - in my IDE the tabs are set up correctly.